### PR TITLE
doc improvement: formatting of UI elements

### PR DESCRIPTION
**Pull request by peknis**
This is mostly internal doc team stuff on the use of
the guilabel tag. However, some of the Bluetooth
samples missed the information of which nRF Connect
for Desktop app to use for testing. Added this in a few samples.

Ref: NCSDK-16137

Signed-off-by: Pekka Niskanen <pekka.niskanen@nordicsemi.no>

### CODE REVIEW

**Review by coderbyheart — APPROVED** [review]

**Review by sebastiandraus — APPROVED** [review]

**Review by VeijoPesonen — APPROVED** [review]

**Review by KAGA164 — APPROVED** [review]
Looks good

**Review by simensrostad — APPROVED** [review]

**Review by tomchy — APPROVED** [review]

File: samples/bluetooth/central_and_peripheral_hr/README.rst [hunk 1]
```diff
@@ -107,13 +107,14 @@ Testing with nRF Connect for Desktop
...
-#. Start `nRF Connect for Desktop`_ and select the connected dongle that is used for communication.
+#. Start `nRF Connect for Desktop`_.
+#. Open the Bluetooth Low Energy app and select the connected dongle that is used for communication.
 #. Open the :guilabel:`SERVER SETUP` tab.
 #. Click the dongle configuration and select :guilabel:`Load setup`.
 #. Load the :file:`hr_service.ncs` file that is located under :file:`samples/bluetooth/central_and_peripheral_hr` in the |NCS| folder structure.
 #. Click :guilabel:`Apply to device`.
 #. Open the :guilabel:`CONNECTION MAP` tab.
-#. Click the dongle configuration(gear ikon) and select :guilabel:`Advertising setup`.
+#. Click the dongle configuration(gear icon) and select :guilabel:`Advertising setup`.
```
Comment (greg-fer) [inline samples/bluetooth/central_and_peripheral_hr/README.rst:117]:
```suggestion
#. Click the dongle configuration (gear icon) and select :guilabel:`Advertising setup`.
```
Missing space?

File: samples/nrf9160/lte_ble_gateway/README.rst [hunk 2]
```diff
@@ -58,20 +58,20 @@ When the connection is established, set switch 2 to **N.C.** to send GNSS data t
...
     .. figure:: /images/nrf_cloud_led_states.svg
        :alt: Application state indicated by LEDs
 
 All LEDs (1-4):
-    * Blinking in groups of two (LED 1 and 3, LED 2 and 4): Modem fault.
-    * Blinking in a cross pattern (LED 1 and 4, LED 2 and 3): Communication error with nRF Cloud.
-    * Blinking in groups of two (LED 1 and 2, LED 3 and 4): Other error.
+    * Blinking in groups of two (**LED 1** and **3**, **LED 2** and **4**): Modem fault.
+    * Blinking in a cross pattern (**LED 1** and **4**, **LED 2** and **3**): Communication error with nRF Cloud.
+    * Blinking in groups of two (**LED 1** and **2**, **LED 3** and **4**): Other error.
```
Comment (greg-fer) [inline samples/nrf9160/lte_ble_gateway/README.rst:74]:
```suggestion
    * Blinking in groups of two (**LED 1** and **LED 3**, **LED 2** and **LED 4**): Modem fault.
    * Blinking in a cross pattern (**LED 1** and **LED 4**, **LED 2** and **LED 3**): Communication error with nRF Cloud.
    * Blinking in groups of two (**LED 1** and **LED 2**, **LED 3** and **LED 4**): Other error.
```
Or:
"(LEDs **3** and **4**)"?

File: samples/nrf9160/lte_ble_gateway/README.rst [hunk 3]
```diff
@@ -129,12 +129,12 @@ After programming the main controller with the sample, test it by performing the
 
 #. Observe that the message ``LTE Sensor Gateway sample started`` is shown in the terminal window, to ensure that the application has started.
 #. The nRF9160 DK now connects to the network. This might take several minutes.
-#. Observe that LED 3 starts blinking as the connection to nRF Cloud is established.
+#. Observe that **LED 3** starts blinking as the connection to nRF Cloud is established.
 #. The first time you start the sample the device will be paired to your account.
-#. Observe that LED 4 is turned on to indicate that the connection to nRF Cloud is established.
-#. Select :guilabel:`Device Management` in the left pane and select :guilabel:`Devices` in the nRF Cloud portal.
+#. Observe that **LED 4** is turned on to indicate that the connection to nRF Cloud is established.
+#. In the nRF Cloud portal, Select :guilabel:`Device Management` in the left pane and select :guilabel:`Devices`.
```
Comment (greg-fer) [inline samples/nrf9160/lte_ble_gateway/README.rst:135]:
```suggestion
#. In the nRF Cloud portal, select :guilabel:`Device Management` in the left pane and select :guilabel:`Devices`.
```

**Pull request merged**